review-tools/ghmerge

```diff
@@ -12,7 +12,7 @@ Option style arguments:
 --web               Merge a web PR (rather than openssl PR)
 --remote <remote>   Repo to merge with (rather than git.openssl.org), usually 'upstream'
 --ref <branch>      Branch to merge with (rather than current branch), usually 'master'
---cherry-pick       Use cherry-pick (rather than pull --rebase)
+--cherry-pick <n>   Cherry-pick the last n commits (rather than rebasing)
```

Comment:
If anyone was using --cherry-pick without the <n> parameter they will have to start putting it in there. I do not know if we care about that. @mattcaswell @paulidale @levitte

Reply:
I could also make `n=1` the default if `--cherry-pick` is not followed by number (i.e., sequence of decimal digits).

Reply:
> I could also make `n=1` the default if `--cherry-pick` is not followed by number (i.e., sequence of decimal digits).

Good idea.

Reply:
Perhaps even if the number after it is larger than 20 or something

Reply:
`ghmerge --nobuild --ref openssl-3.0 --cherry-pick 16051 paulidale` - otherwise this won't work.

Reply:
Yeah, even better to limit `n` in order to prevent clashes with any given commit ID.
Done so, allowing (to be on the safe side) `1 <= n <= 99`.